bluesky/run_engine.py

```diff
@@ -415,13 +421,41 @@ def request_pause(self, defer=False, name=None, callback=None):
                 print("Cannot pause from {0} state. "
                       "Ignoring request.".format(self.state))
 
-    def _register_scan_callback(self, name, func):
-        """Register a callback to be processed by the scan thread.
+    def _subscribe_lossless(self, name, func):
```

Comment:
Yeah, sorry, it was added in a later commit. You can see my line of thought in the commit messages... "Well as long as I'm touching `register_mds.py` to fix `insert_descriptors` I might as well fix this thing...."